### ISSUE TITLE
store exemplars

### ISSUE THREAD

**Issue by aooating** [akshitac8/OW-DETR#5]
Where in the code do we store 50 samples per known category?

**Comment by akshitac8** [comment]
Hello @aooating We don't store 50 samples per category for OW-DETR.

**Comment by aooating** [comment]
In C. Additional Implementation Details, there is writing in the incremental step the learning rate is reduced by a factor of 10 and trained using a set of 50 stored exemplars per known class.

**Comment by akshitac8** [comment]
The 50 stored exemplars are the images used for fine-tuning which are present in _ft.txt files.

**Comment by aooating** [comment]
Thank you for your reply! I would also like to know how to generate those ft.txt files. Looking forward to your reply!

**Comment by akshitac8** [comment]
@aooating the files are already present inside these paths
https://github.com/akshitac8/OW-DETR/tree/main/data/OWDETR/VOC2007/ImageSets/
https://github.com/akshitac8/OW-DETR/tree/main/data/OWOD/VOC2007/ImageSets
